### ISSUE TITLE
Missing Notion Rollup functionality

### ISSUE THREAD

**Issue by debanV** [NotionX/react-notion-x#174]
Rollup functionality not working, so maybe it's not developed, I don't find in the documentation.

**Comment by normdoow** [comment]
Yeah, its not really supported. I think it works for one row?

**Comment by debanV** [comment]
No. It's not working at all. Nothing appear in span class="notion-property notion-property-rollup"

**Comment by transitive-bullshit** [comment]
It's not currently supported.

There are a bunch of advanced database features that would take a lot of time and effort to emulate faithfully (like eval properties).

PRs are welcome 😄
